javatests/arcs/core/policy/PolicyTest.kt

```diff
@@ -0,0 +1,45 @@
+package arcs.core.policy
+
+import arcs.core.data.Annotation
+import arcs.core.data.Capabilities
+import arcs.core.data.Capability.Encryption
+import arcs.core.data.Capability.Persistence
+import arcs.core.data.Capability.Ttl
+import arcs.core.policy.PolicyRetention
+import arcs.core.policy.PolicyTarget
+import arcs.core.policy.StorageMedium
+import com.google.common.truth.Truth.assertThat
+import java.util.concurrent.TimeUnit
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import kotlin.test.assertFailsWith
+
+@RunWith(JUnit4::class)
+class PolicyTest {
+    @Test
+    fun policyTarget_toCapabilities() {
+        val policyTarget = PolicyTarget(
+            schemaName = "schema",
+            maxAgeMs = TimeUnit.MILLISECONDS.convert(2L, TimeUnit.HOURS),
+            retentions = listOf(
+                PolicyRetention(medium = StorageMedium.DISK, encryptionRequired = true),
+                PolicyRetention(medium = StorageMedium.RAM, encryptionRequired = false)
+            ),
+            fields = emptyList(),
+            annotations = emptyList()
+        )
+        val capabilities = policyTarget.toCapabilities()
+        assertThat(capabilities).hasSize(2)
+        assertThat(
+            capabilities[0].containsAll(
+                Capabilities(listOf(Persistence.ON_DISK, Encryption(true), Ttl.Minutes(120)))
+            )
+        )
+        assertThat(
+            capabilities[0].containsAll(
+                Capabilities(listOf(Persistence.IN_MEMORY, Ttl.Minutes(120)))
+            )
+        )
```

Comment:
This seems wrong. Should the second one be `capabilities[1]`?

Also, you need to follow up `assertThat(...)` with something (`.containsAll`? `.isTrue`?)

Also, I think this whole thing should actually be:

```
assertThat(capabilities).containsExactly(
    Capabilities(listOf(...)),
    Capabilities(listOf(...))
).inOrder()
```

Reply:
oh my, this uncovered a lot of problems, including missing `isEquivalent` method in KT and broken `isEquivalent` method in TS. fixed, added tests. 
Thanks for the catch!

Reply:
Hooray for code reviews!

Reply:
oh yeah!! 👍 💯